src/stylesheets/core/mixins/_add-aspect.scss

```diff
@@ -0,0 +1,18 @@
+@mixin add-aspect($ratio...) {
+  $ratio: unpack($ratio);
+  $important: null;
+  @if has-important($ratio) {
+    $ratio: nth($ratio, 1);
+    $important: "!important";
+  }
```

Comment:
Added some code here to enable a `!important` flag on these, similar to https://github.com/uswds/uswds/blob/eb518a32de08caa21415879b9de7a27324c40b6d/src/stylesheets/core/mixins/utilities/_left.scss#L3